webrender/src/clip_scroll_node.rs

```diff
@@ -198,7 +198,7 @@ impl ClipScrollNode {
         LayerSize::new(overscroll_x, overscroll_y)
     }
 
-    pub fn set_scroll_origin(&mut self, origin: &LayerPoint) -> bool {
+    pub fn set_scroll_origin(&mut self, origin: &LayerPoint, clamp_to_bounds: bool) -> bool {
```

Comment:
nit: Do you think it makes sense to have a `BoundsClampingMode` enum instead? Perhaps it's not too worth it, so just a suggestion :)